src/v2/Apps/Show/Components/ShowInfoLocation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowInfoLocation_show } from "v2/__generated__/ShowInfoLocation_show.graphql"
+
+interface ShowInfoLocationProps {
+  show: ShowInfoLocation_show
+}
+
+export const ShowInfoLocation: React.FC<ShowInfoLocationProps> = ({ show }) => {
+  // Merge show location and fair location into a single object, favoring show
+  const location = Object.entries(show.location).reduce((acc, [key, value]) => {
+    return { ...acc, [key]: value ?? show.fair?.location?.[key] }
+  }, {} as typeof show.location)
+
+  const lines = [
+    location.display,
```

Comment:
Apparently, the creatively named `display` field contains booth information if the show is in a fair.

Reply:
Show/fair modeling has about 10 different fields with `start` and `display` in them and it's almost impossible to figure out what we want to be happening there.